cross-chain/arbitrum/external/goerliArbitrum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0x23908A62110e21C04F3A4e011d24F901F911744A",
```

Comment:
I [confirmed the address is correct](https://book.wormhole.com/reference/contracts.html) but where does the ABI come from? It looks like this contract [was not verified](https://goerli.arbiscan.io/address/0x23908A62110e21C04F3A4e011d24F901F911744A#code). If we can't verify the ABI maybe we should leave just an address here?

Reply:
I copied ABI from Arbiscan. But agree, we can leave the address only. [39c830d](https://github.com/keep-network/tbtc-v2/pull/554/commits/39c830ded61a9fdc92afcf0a7a6b02624e79bbf3)

Reply:
What's more the contracts verified on the `arbitrumOne` network are upgradable contracts. So there is no point in adding ABIs of proxy contracts, and we don't want to maintain the ABIs of implementations.